CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8.11)
```

Comment:
In the [Readme](https://github.com/tangrams/tangram-es#build) we have mentioned cmake min is 3.0.